.github/workflows/build-deploy-ferceqr.yml

```diff
@@ -0,0 +1,107 @@
+---
+name: build-deploy-ferceqr
+on:
+  workflow_dispatch:
+
+env:
+  GCP_BILLING_PROJECT: ${{ secrets.GCP_BILLING_PROJECT }}
+  GCS_OUTPUT_BUCKET: gs://builds.catalyst.coop
+  BATCH_JOB_JSON: batch_job.json
+
+jobs:
+  build_and_deploy_eqr:
```

Comment:
At some point all of the docker build stuff should be refactored into a reusable workflow.